package.json

```diff
@@ -33,6 +33,7 @@
   },
   "devDependencies": {
     "buble": "^0.19.7",
+    "proxyquire": "^2.1.0",
```

Comment:
did you mean to add proxyquire here?

Reply:
I did, because it is used in a new test - which I missed in the `git add`. It's now part of the PR. See 2d48ed2.